ttt.py

```diff
@@ -146,7 +146,7 @@ def transcribe_deepgram(calljson, audiofile):
 def send_notifications(calljson, destinations):
     # Scrubadub redacts PII let's try and clean the text before
     # goes out the door
-    body = scrubadub.clean(calljson["text"])
+    body = scrubber.clean(calljson["text"])
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (docstrings):** Please update the docstring for function: `send_notifications`

Reason for update: The method used for cleaning text has changed from 'scrubadub.clean' to 'scrubber.clean'.

Suggested new docstring:
```
# Scrubber is used to clean the text before it goes out the door, with adjustments for less aggressive email redaction.
```